index.js

```diff
@@ -177,3 +221,45 @@ function findBestSubtitleFile(list) {
     .filter((x) => x.subs)
     .sort((x, y) => x.priority > y.priority)[0];
 }
+
+function processFormats(formats) {
+  const filteredFormats = formats.filter(({ format_id, fps, height, vcodec }) => format_id !== 'source' && !format_id.startsWith('http') && vcodec && vcodec !== 'none' && !vcodec.includes('av01') && !vcodec.includes('vp9') && !vcodec.includes('vp09') && ((height >= 1080 && fps <= 30) || (height < 1080)))
+  .sort((prevFormat, nextFormat) => nextFormat.tbr - prevFormat.tbr);
+
+  const selected4K = filteredFormats.find((format) => (format.height === 2160 && format.acodec !== 'none')) || filteredFormats.find((format) => format.height === 2160);
```

Comment:
isn't this line of code just equivalent to:

`const selected4K = filteredFormats.find((format) => format.height === 2160);`

?

Reply:
So I still have the combined format prioritised. So it tries to find the combined track and if it can't it falls back to just getting the first video track for that format. Therefore it acts sort of like a fallback.